contracts/script/examples/cross-token/Makefile

```diff
@@ -0,0 +1,59 @@
+# Makefile for USDC Token Bridge on Calibration Network
+
+# Include environment variables from .env file
+include .env
+export
+
+.PHONY: help deploy-usdctest mint-usdc check-balance deploy-replica deploy-bridge update-replica deposit-usdc withdraw-usdc
+
+help:
+	@echo "Available targets:"
+	@echo "  deploy-usdctest  - Deploy the USDCTest contract on the Calibration Network."
+	@echo "  mint-usdc        - Mint 1000 USDCTest tokens to the wallet specified in the .env file."
+	@echo "  check-balance    - Check the balance of USDCTest tokens in the wallet."
+	@echo "  deploy-replica   - Deploy the token replica contract on the subnet."
+	@echo "  deploy-bridge    - Deploy the token bridge contract on the Calibration Network."
+	@echo "  update-replica   - Update the replica with the token bridge controller's address."
+	@echo "  deposit-usdc     - Deposit USDC to the subnet."
+	@echo "  withdraw-usdc    - Withdraw USDC from the subnet."
+	@echo ""
+	@echo "Usage:"
+	@echo "  make <target>    - Run a specific target."
+	@echo "  make help        - Display this help message."
+
+deploy-usdctest:
+	@echo "Deploying USDCTest contract..."
+	forge script DeployUSDCTest.s.sol --broadcast --rpc-url $$CALIBNET_RPC_URL --private-key $$PRIVATE_KEY --skip-simulation
+
+mint-usdc:
+	@echo "Minting 1000 USDCTest tokens..."
+	cast send $$USDC "$$(cast calldata "mint(uint256)" 1000)" --rpc-url $$CALIBNET_RPC_URL --private-key $$PRIVATE_KEY
```

Comment:
```suggestion
	cast send $$USDC "$$(cast calldata "mint(uint256)" 1000000000000000000000)" --rpc-url $$CALIBNET_RPC_URL --private-key $$PRIVATE_KEY
```

Reply:
ERC20 uses 18 decimals by default, so minting 1000 USDC requires passing 1000000000000000000000 to mint.

Reply:
This is now set in the environment variable. This should resolve your concern as the user can choose any number